src/styles/user-signup.css

```diff
@@ -0,0 +1,66 @@
+.SignUpForm {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+  }
+
+  .SignUpForm div {
+    display: flex;
+    flex-direction: column;
+    text-align: left;
+  }
+
+  .SignUpForm p {
+    font-size: 12px;
+    text-align: center;
+  }
+
+  .SignUpForm ::placeholder {
+    color: #989898;
+  }
+  
+  .SignUpForm form {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+  
+  .SignUpForm label {
+    margin: 10px 0;
+    padding-top: 20px;
+  }
+
+  .SignUpForm img {
+    width: 200px;
+    padding-top: 20px;
+    margin: auto;
```

Comment:
Theres no such thing as `top: auto` or `bottom: auto` which is in effect what you have here. Id replace with this:
```
    margin: 0 auto;
```